mii/grpc_related/modelresponse_server.py

```diff
@@ -26,13 +26,29 @@ def _unpack_proto_query_kwargs(self, query_kwargs):
         }
         return query_kwargs
 
+    def _get_model_time(self, model, sum_times=False):
+        model_times = []
+        if hasattr(model, "model_times"):
+            model_times = model.model_times()
+
+        if len(model_times) == 1:
+            model_time = model_times[0]
+        elif len(model_times) > 1:
+            if sum_times:
+                model_time = sum(model_times)
+            else:
+                model_time = model_times[-1]
+        else:
+            model_time = -1
+        return model_time
```

Comment:
Maybe add comments to help understand this. Why would model_times sometimes have len==1 and sometimes have multiple entries? Also what is the significance of returning -1 when len(model_times)==0?